src/main/resources/detect-sh.sh

```diff
@@ -119,8 +129,22 @@ get_detect() {
   fi
 }
 
+set_detect_java_path() {
+if [ -n "${DETECT_JAVA_PATH}" ]; then
+    echo "Java Source: DETECT_JAVA_PATH=${DETECT_JAVA_PATH}"
+  elif [ -n "${JAVA_HOME}" ]; then
+    DETECT_JAVA_PATH="${JAVA_HOME}/bin/java"
+     echo "Java Source: JAVA_HOME=${DETECT_JAVA_PATH}"
```

Comment:
Definitely a nit, but I think the output of this echo might imply a problem ("Detect thinks JAVA_HOME points to my java executable instead of JAVA_HOME") where the in fact is no problem. Maybe something like?:
echo "Java Source: JAVA_HOME/bin/java=${DETECT_JAVA_PATH}

Reply:
You're right. Fixing now.